packages/lake/src/components/TabView.tsx

```diff
@@ -32,20 +42,28 @@ import { FocusTrap } from "./FocusTrap";
 import { Icon, IconName } from "./Icon";
 import { LakeText } from "./LakeText";
 import { Link } from "./Link";
-import { PressableText } from "./Pressable";
+import { PressableText, PressableTextProps } from "./Pressable";
 import { Space, SpacingValue } from "./Space";
 import { TransitionView } from "./TransitionView";
 
 const TABS_HEIGHT = 40;
 export const tabsViewHeight = TABS_HEIGHT + 1;
 
-type Tab = {
-  label: string;
-  url: string;
-  icon?: IconName;
-  withSeparator?: boolean;
-  count?: number;
-};
+type Tab =
```

Comment:
Quick tip: using intersection types, you can simplify this too:

```ts
type Tab = ({ id: string } | { url: string }) & {
  count?: number;
  icon?: IconName;
  label: string;
  withSeparator?: boolean;
};
```

More infos: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html